CHANGELOG.md

```diff
@@ -10,11 +10,23 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Breaking changes
+
+- Updated OpenTelemetry to v0.102.1. (@mattdurham)
```

Comment:
```suggestion
- Updated OpenTelemetry to v0.102.0. (@mattdurham)
```
I'd even omit the whole "Updated OpenTelemetry" line, for two reasons:
* The upgrade as a whole is not the breaking change.
* The upgrade is already mentioned below.

Reply:
It is OpenTelemetry v0.102.1, Contrib did not need a change so its v0.102.0.